### PR TITLE
AWS Picchu Edit page, images & license.

### PR DESCRIPTION
**Pull request by jminor**
This PR adds a new page for the AWS Picchu Edit. It does not yet have proper links to the downloads. I will update it when those are available.

### CODE REVIEW

**Comment by jminor** [comment]
Okay, I fixed the problems I had earlier, and updated with the real links to the files. This is ready for review.

**Comment by jminor** [comment]
Also: I recommend "Squash and Merge" when this is approved.

File: src/component/Table.astro [hunk 1]
```diff
@@ -21,11 +21,11 @@ const { downloadData } = Astro.props;
           <td class="pl-2 pt-3">
             <Button text={item.buttonText} type={item.type ? item.type : 'primary'} url={item.downloadUrl} />
           </td> 
-          <td class="pl-4">{item.size}</td>   
           <td class="pl-4">
             {item.description} {item.descriptionBold ? 
               ( <text><b>{item.descriptionBold}</b>{item.extraDescription}</text> ) : null} 
           </td>
+          <td class="pl-4"><p>{item.size}</p><p class="hash">{item.hashA}</p><p class="hash">{item.hashB}</p></td>
```
Comment (ee33) [inline src/component/Table.astro:28]: Dumb q, what does it do for items without hashA or hashB? (Whitespace fine with me, error message less so...)

File: src/component/Table.astro [hunk 2]
```diff
@@ -44,7 +44,11 @@ const { downloadData } = Astro.props;
...
   }
-  tr td:nth-child(2){
-    width: 125px;
+  tr td:nth-child(3){
+    width: 330px;
   }
-</style>
\ No newline at end of file
+  .hash {
+   font-weight: 300;
```
Comment (ee33) [inline src/component/Table.astro:51]: OK. Maybe someday we'll make them roll-over text or something.

File: src/content/articles/aws-picchu-edit.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  licenseButtonText: "ASWF Asset License",
+  licenseButtonLink: "aws-picchu-edit/aws-picchu-edit-license",
+  # This header is only if the table needs a header < Please see Intel page for example of that >
+  downloadTableHeader: "",
+
+  # The download links and button setup for the download table.
+  downloads: [{
+    buttonText: "DOWNLOAD",
+    downloadUrl: "https://dpel-assets.aswf.io/picchu/picchu_v10_az.dra.zip",
+    size: "3.72 GB",
```
Comment (ee33) [inline src/content/articles/aws-picchu-edit.md:98]: Chrome seems to think this file is 2.9 GB, and the other file sizes don't match either. Maybe out of date?

**Review by ee33 — COMMENTED** [review]
Double-check the file sizes and let us know - thanks.
(I didn't build the website... nor check the hashes...)

File: src/component/Table.astro (hunk 1 above)
Comment (jminor) [reply src/component/Table.astro:28]: The pages without hashes look like this:
image

File: src/component/Table.astro (hunk 2 above)
Comment (jminor) [reply src/component/Table.astro:51]: Maybe a link to instructions for how to verify based on those hashes? I'll leave that for a future PR.

**Comment by jminor** [comment]
Fixed download sizes (those were from an earlier revision of the assets). Thanks for catching that!

File: src/content/articles/aws-picchu-edit.md (hunk 3 above)
Comment (jminor) [reply src/content/articles/aws-picchu-edit.md]: Fixed.

**Review by ee33 — APPROVED** [review]

**Comment by ee33** [comment]
"Squash and merge" is "not enabled for this repo" ... hope that's OK

**Pull request merged**